test/cockroachdb_test.go

```diff
@@ -0,0 +1,82 @@
+package helm_testing
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/stretchr/testify/require"
+	rbacv1 "k8s.io/api/rbac/v1"
+)
+
+func Test_Role(t *testing.T) {
+	helmChartPath, err := filepath.Abs("../cockroachdb")
+	require.NoError(t, err)
+	require.NotNil(t,helmChartPath)
+
+	t.Run("Test TLS Enabled but certs not provided", func(t *testing.T) {
```

Comment:
We need to validate this in combination with the k8s version - this will work on older versions of k8s with the v1beta1 api but will fail on newer versions where the legacy unknown signer isn't available.  So this test needs to be more specific